plugins/plugin-vue/test/__snapshots__/plugin-vue-ts-tsx-jsx.test.js.snap

```diff
@@ -2,10 +2,6 @@
 
 exports[`plugin vue with jsx 1`] = `
 Object {
-  ".css": Object {
-    "code": "",
-    "map": "",
-  },
```

Comment:
Hiding right under our noses! 🎉

Reply:
aha! Glad we have the individual test files now 👍